Extension/src/LanguageServer/client.ts

```diff
@@ -682,7 +682,7 @@ export interface Client {
     handleConfigurationEditUICommand(viewColumn?: vscode.ViewColumn): void;
     handleAddToIncludePathCommand(path: string): void;
     handleGoToDirectiveInGroup(next: boolean): Promise<void>;
-    handleGenerateDoxygenComment(codeActionArguments: DoxygenCodeActionCommandArguments | undefined): Promise<void>;
+    handleGenerateDoxygenComment(args: DoxygenCodeActionCommandArguments | vscode.Uri | undefined): Promise<void>;
```

Comment:
in the case where the uri is passed, do we not get the true args in a second parameter?

Reply:
The Uri is passed when using the context menu.  The DoxygenCodeActionCommandArguments are passed when a CodeAction is used.  There is no scenario in which both are passed.